### ISSUE TITLE
extension for monocular

### ISSUE THREAD

**Issue by tomprimozic** [MAC-VO/MAC-VO#3]
Hi, do you have any thoughts or ideas on how could we leverage this technique for monocular VO?

**Comment by haleqiu** [comment]
@tomprimozic 
Yes, we are planning on this. As we stated in the Conclusion of our paper, we are also interested in integrating IMU with current framework. 

If anyone is interested in collaborating on this research, we would be happy to discuss potential opportunities!
